packages/scrollable-video/src/components/video.js

```diff
@@ -93,6 +93,34 @@ function removeListenerFromMQList(mql, listener) {
   }
 }
 
+/**
+ *
+ * On iOS low power mode, autoplaying video is not allowed.
+ * Also, video cannot be played unless user interact with the device in such situation.
+ * This hack plays video on user first touch for hiding the playback button UI when it cannot be autoplayed.
+ */
+function playVideoWhenAutoplayIsDisabledOnTouch() {
+  if (typeof window === 'undefined') return
+  const playVideoWhenAutoplayIsDisabled = () => {
+    const videos = document.querySelectorAll('video')
+    if (videos && videos.length > 0) {
+      videos.forEach(video => {
+        if (!video.playing) {
+          // video should be autoplayed.
+          // if video is not playing, play video now.
+          const playPromise = video.play()
+          playPromise.catch(() => {
```

Comment:
Would we add some error log here for debugging in the future ?
(ex. console.error)

Reply:
Updated in https://github.com/twreporter/orangutan-monorepo/pull/110/commits/d4cba91073c3a628ccad2216edd0b90ef4b14a0b.